CONTRIBUTING.md

```diff
@@ -172,6 +172,22 @@ phenv pip uninstall stubs
 This app is built on top of several modules developed by Phantom. Since we don't have access to the source for these
 modules, we've stubbed them out in the `phantom` directory so that they can be imported by the test code.
 
-# TODO
+# Testing the App
 
-When we implement actions, describe how to create dummy events in the UI and test the actions via a playbook.
+First, get alerts into Phantom by going to your configured asset's Ingest Settings and enabling
+polling. Then, hit `Poll Now`. You likely will want to increase the `Maximum Containers` and `Maximum Artifacts`
+properties to ingest more alerts.
+
+Go to the Sources tab and look at the newly ingested alerts. Use the "Action" button and select from the action you
+wish to run. Additionally, use the "Playbook" button to run a playbook.
+
+## Reset Polling Checkpoint
```

Comment:
Might be a good `make` command...
`make clear-checkpoint`

Reply:
If automated, it will either just have to be a script or use an environment variable to know the asset number to reset rather than `make clear-checkpoint` alone).

Reply:
Maybe another PR?

This seems to do the check:

```
#!/bin/bash

APP_ID="4d8f53a7-7b12-4d7d-8b01-6575680acf6f"

main() {
  asset_number="${1:?Missing parameter 'asset_number' at position 1}"
  # TODO: add `/` in front of opt
  state_file="opt/phantom/local_date/app_states/${APP_ID}/${asset_number}_state.json"

  if [[ -f "${state_file}" ]]; then
    state_json=$(cat $state_file)
    current_value=${state_json:13}
    current_value=${current_value::${#current_value}-1}
    sed -i '' "s/${current_value}/0/g" $state_file
    cat $state_file
  else
    echo "No asset with number ${asset_number}"
  fi
}

main "$@"

```